src/app/portlets/dot-edit-content/dot-edit-content.component.ts

```diff
@@ -154,6 +157,27 @@ export class DotEditContentComponent implements OnInit {
         });
     }
 
+    /**
+     * Hanlde changes in the configuration of "View As" toolbar
+     *
+     * @param {DotEditPageViewAs} viewAsConfig
+     * @memberof DotEditContentComponent
+     */
+    changeViewAsHandler(viewAsConfig: DotEditPageViewAs): void {
+        // TODO: Make a new request ?? to this.editPageService.getAs('4', viewAsConfig.persona, viewAsConfig.language)
+        console.log(viewAsConfig);
```

Comment:
why don't you call the this.editPageService.getAs?

Reply:
Yeah, we should call it and pass the data, no matter it will do the same request.